runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -156,6 +156,8 @@ public class ItemChargePlugin extends Plugin
 	private static final int MAX_BLOOD_ESSENCE_CHARGES = 1000;
 	private static final int MAX_BRACELET_OF_CLAY_CHARGES = 28;
 
+	private static final int TRAHAEARN_MINE_REGION = 13249;
```

Comment:
Rather than adding this region to check, split the normal and soft clay message checks into two, and run the inventory check code only for the `BRACELET_OF_CLAY_USE_TEXT_TRAHAEARN` branch.